zarf.schema.json

```diff
@@ -552,7 +585,7 @@
           "description": "The name of the resource or selector to wait for",
           "examples": [
             "podinfo",
-            "app\u0026#61;podinfo"
+            "app&#61;podinfo"
```

Comment:
These changes are going to break the website since they will display funny

Reply:
ah ok no you're right - the \u was to get around limitations in the jsonschema library but looks like `jq` did this change which is after that part of the process